packages/security/src/store/index.ts

```diff
@@ -18,11 +18,11 @@ const state: SecurityModel = {
   groupRights: { user: null, anonymous: null, roles: [] }
 }
 
-Vue.use(Vuex)
 export default new Vuex.Store({
+  state,
+  mutations,
   actions,
   getters,
-  mutations,
-  state,
-  strict: process.env.NODE_ENV !== 'production'
+  modules: {
```

Comment:
Lege modules definitie?

Reply:
ja is de default cli setup , vind ik zelf wel mooi geeft aan dat je het kunt gaan gebruiken